### PR TITLE
testing of UI connectivity to VMODL API - all enpoints are testing ok

### PR DESCRIPTION
**Pull request by marksoper**

### CODE REVIEW

**Comment by msterin** [comment]
is there a way to avoid copy0n-paste for each argsSOAP construction ? 
The rest looks good to my untrained eye

**Comment by marksoper** [comment]
msterin: good call, I've made it more DRY, those changes will be in the next PR

**Comment by msterin** [comment]
> msterin: good call, I've made it more DRY, those changes will be in the next PR

LGTM then. Looking forward for the next PR :-)

File: ui/scripts/controllers/dvol-main.js [hunk 1]
```diff
@@ -16,12 +16,63 @@ define([], function() {
     // TODO: remove this
     //
 
-    DvolVsanService.getTenants()
-    .then(function(res) {
-      console.log('getTenants: ' + res);
-    }, function(err) {
-      console.log('ERROR getTenants: ' + err);
-    });
+    var mockTenant = {
```
Comment (pdhamdhere) [inline ui/scripts/controllers/dvol-main.js]: Should tests be separate test file?

**Pull request closed without merging**